### PR TITLE
OSDOCS#11238: Add relnote for etcd certificate automatic rotation

### PR DESCRIPTION
**Pull request by tmalove**
Version(s):
4.17

Issue:
[OSDOCS-11238](https://issues.redhat.com/browse/OSDOCS-11238)

Link to docs preview:
[Security](https://80746--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes#ocp-4-17-etcd-certificates_release-notes) Updated: 08/29/2024

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by tmalove** [comment]
@tjungblu  I updated the content per your suggestion.
@geliu2016 Will you review this relnote entry?

Thanks!

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -503,6 +503,17 @@ The Network Observability Operator releases updates independently from the {prod
...
+[id="ocp-17-auto-rotate-etcd-ca"]
+==== Automatic rotation of signer certificates and a new namespace
+
+With this release, all `etcd` certificates originate from a new namespace: `openshift-etcd`. When a new signer certificate is close to its expiration date, the following actions occur:
+
+* An automatic rotation of the signer certificate activates.
+* The certificate bundle updates.
+* All certificates regenerate with the new signers.
+
+Manual rotation of signer certificates is still supported by deleting the specific secret and waiting for the status pod rollout to complete.
```
Comment (geliu2016) [inline release_notes/ocp-4-17-release-notes.adoc:515]: as I though, "waiting for the status pod rollout to complete" updated to "waiting for cluster operator status become to health", @tjungblu, please correct me if it's not exact, thanks

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -503,6 +503,17 @@ The Network Observability Operator releases updates independently from the {prod
 [id="ocp-4-17-etcd-certificates_{context}"]
 === Security
 
+[id="ocp-17-auto-rotate-etcd-ca"]
+==== Automatic rotation of signer certificates and a new namespace
```
Comment (tjungblu) [inline release_notes/ocp-4-17-release-notes.adoc:507]:
```suggestion
==== Automatic rotation of signer certificates
```

The namespace is already there, so let's not have that in the title

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (tjungblu) [reply release_notes/ocp-4-17-release-notes.adoc:515]: I think it means the same :)

Comment (geliu2016) [reply release_notes/ocp-4-17-release-notes.adoc:515]: ok, thanks

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -503,6 +503,17 @@ The Network Observability Operator releases updates independently from the {prod
 [id="ocp-4-17-etcd-certificates_{context}"]
 === Security
 
+[id="ocp-17-auto-rotate-etcd-ca"]
+==== Automatic rotation of signer certificates
+
+With this release, all `etcd` certificates originate from a new namespace: `openshift-etcd`. When a new signer certificate is close to its expiration date, the following actions occur:
+
+* An automatic rotation of the signer certificate activates.
```
Comment (dfitzmau) [inline release_notes/ocp-4-17-release-notes.adoc:511]: These actions can occur in any order? If not, would numbered steps work?

File: release_notes/ocp-4-17-release-notes.adoc [hunk 4]
```diff
@@ -503,6 +503,17 @@ The Network Observability Operator releases updates independently from the {prod
 [id="ocp-4-17-etcd-certificates_{context}"]
 === Security
 
+[id="ocp-17-auto-rotate-etcd-ca"]
```
Comment (dfitzmau) [inline release_notes/ocp-4-17-release-notes.adoc:506]:
```suggestion
[id="ocp-17-auto-rotate-etcd-ca_{context}"]
```

Example from the 4.16 release notes: 
Screenshot from 2024-08-30 10-51-04

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**